arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/Hub.java

```diff
@@ -361,36 +362,36 @@ public List<Module> getModulesToCount() {
     return Stream.concat(
             Stream.of(
                 this,
-                romLex,
                 add,
                 bin,
                 blakeModexpData,
                 blockdata,
                 blockhash,
-                ext,
                 ecData,
+                exp,
+                ext,
```

Comment:
Can we add GAS too ? It's empty now, but at least coordinator won't have to change it's setting again and again